### PR TITLE
Decrease window to fetch blocks at IDL time from 500 to 100

### PR DESCRIPTION
**Pull request by dahlia**
At the moment, I only decreased the hard-coded number.  I will add an option to configure this in the next patch.

Also added a small hook script to check duplicated link definitions in *CHANGES.md* docs.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -27,7 +27,12 @@ To be released.
 
 ### Behavioral changes
 
+ -  `Swarm<T>` became to send 100 blocks (instead of 500 blocks) for each
+    reply during IDL, in order to stabilize connection in high latency
+    environments.
```
Comment (earlbread) [inline CHANGES.md]: We need to add the PR number and link.

**Comment by dahlia** [comment]
As this ad-hoc adjustment still doesn't work well, I'm going to work further.

**Comment by dahlia** [comment]
> As this ad-hoc adjustment still doesn't work well, I'm going to work further.

I'm going to work further in another pull request.

@earlbread @longfin @limebell @moreal Please review this again.

**Comment by moreal** [comment]
@dahlia Tests seem to fail. 😢

**Pull request merged**